### ISSUE TITLE
Unsupervised Learning of Visual Features by Contrasting Cluster Assignments 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#13]
# TL;DR
Whereas SimCLR and others require large batch sizes in representation learning, SwAV makes it possible to learn in small batches by online training that train the model to cluster consistently between differently augmented two image . Good results in object detection and image classification using the obtained representations.
![image](https://user-images.githubusercontent.com/24524018/89533151-ec6e4900-d82d-11ea-8ead-094870a7a055.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.09882

# Submission Dates(yyyy/mm/dd)
2020/06/17

# Authors and institutions
Mathilde Caron, Priya Goyal,Ishan Misra, Julien Mairal Piotr Bojanowski, Armand Joulin
- Inria 
- Facebook AI Research 

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/89533161-f132fd00-d82d-11ea-9bb0-39fb85df9f34.png)

# Comments
